examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -44,6 +48,37 @@ class Task extends TypedEventEmitter<ITaskEvents> implements ITask {
             this.emit("priorityChanged");
         });
     }
+    private readonly resetDiffProperties = (): void => {
+        this.diffPriority = DEFAULT_DIFF_PRIORITY;
+        this.diffType = "none";
+        this.emit('externalPriorityChanged');
+        this.diffName = '';
+        this.diffType = "none";
+        this.emit('externalNameChanged');
+    }
+    public externalNameChanged = (savedName: string): void => {
+        this.diffName = savedName;
+        this.diffType = "change";
+        this.emit('externalNameChanged');
+    }
+    public externalPriorityChanged = (savedPriority: number): void => {
+        this.diffPriority = savedPriority;
+        this.diffType = "change";
+        this.emit('externalPriorityChanged');
+    }
+    public acceptChange = (): void => {
```

Comment:
Maybe use a more verbose name just to make it clear what's happening - something like overwriteWithSourceData or something

Reply:
Let me push up my latest changes with what I think are clearer names.